keps/sig-release/1733-release-notes/README.md

```diff
@@ -47,19 +54,19 @@ As this is a somewhat drastic departure from the current process, this should be
 approach:
 
 - Consolidation and Clean-up
-  - Update `anago` to use `release-notes`
-  - Replace the "Detailed Bug Fixes And Changes" section in the release notes with a new
-    website that allows user to filter and search.
-  - Generate a consumable JSON file via `release-notes` as part of the Release Team duties
-  - Identify and stop tracking any "External Dependencies" that a vanilla Kubernetes install
-    does not rely on. (eg. Things in `cluster/addons`)
+  - [x] Update `anago` to use `release-notes`
```

Comment:
Add a checkbox about removing the `relnotes` tool.

Reply:
Yes, changed that point to mention that we use `krel changelog` and added another point for the removal of the `relnotes` script.